src/middleware/expiryValidator.js

```diff
@@ -0,0 +1,17 @@
+const { access_token } = require("../models/Access_tokenModel");
+const jwt = require("jsonwebtoken");
+async function expiryValidator(req, res, next) {
+  try {
+    const token = await access_token.findOne({ token: req.headers.access });
+    if (token) {
+      jwt.verify(token.token, "secret");
```

Comment:
first verify the token by JWT and then check the token in database to get the user details.
or you can store most of details in JWT token.